spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -514,6 +517,20 @@ public void setSize(Integer size) {
 		@NestedConfigurationProperty
 		private final ListenerRetry retry = new ListenerRetry();
 
+		@NestedConfigurationProperty
+		private final SimpleContainer simple = new SimpleContainer();
```

Comment:
So Direct only supports one property? I am a bit confused. The hierarchy of properties are becoming quite complex now so I am wondering if we should create those two namespaces at all.